src/Jackett.Common/Definitions/rockbox.yml

```diff
@@ -38,28 +38,32 @@
     inputs:
       search: "{{if .Query.Artist}}{{ .Query.Artist }}{{else}}{{ .Keywords }}{{end}}"
     rows:
-      selector: body:nth-child(2) tbody tr td:nth-child(2) table.lista tbody tr:nth-child(2) table.lista tbody tr
+      selector: tr:has(a[href^="download.php?id="])
     fields:
       title:
         selector: td a[href^="details.php?id="]
       details:
         selector: td a[href^="details.php?id="]
         attribute: href
-      comments: # link to the comments of the torrent
+      comments:
         selector: td a[href^="details.php?id="]
         attribute: href
       download:
         selector: td a[href^="download.php?id="]
         attribute: href
       date:
-        selector: td:nth-child(6)
-      size: # size of the torrent (units are handled automatically)
+        # 13/10/2018
```

Comment:
where is the first date block I gave you for the timeago?
```yaml
      # dates come in two flavours:
      date:
        # 1 hour ago, 2 days ago, 3 weeks ago etc
        selector: td:nth-child(6):not(:contains("/"))
        optional: true
        filters:
          - name: timeago
```

Reply:
oh, I did not understand that the 2 dates blocs were needed, and would work with each others.

Reply:
yeah, the selector picks which date format best fits the filter, and the optional:true ensure the other block does not produce an error when it fails to match a selector